ext/nokogiri/extconf.rb

```diff
@@ -387,7 +391,16 @@ def using_system_libraries?
   do_clean
 end
 
-RbConfig::MAKEFILE_CONFIG['CC'] = ENV['CC'] if ENV['CC']
```

Comment:
What about leaving this unchanged and put `if openbsd? && !ENV['CC'] ... ENV['CC'] = find_executable 'egcc' end` above it so you could still override `CC`?